lib/components/narrative/tabbed-itineraries.js

```diff
@@ -92,26 +122,34 @@ class TabButton extends Component {
           </span>
           {/* the fare / calories summary line */}
           <span>
-            {minTotalFare > 0 && (
+            {/* Display "No Fare Info" if there is a transit leg w/o fare information */}
+            {transitFareNotProvided ? (
               <>
-                <FormattedMessage
-                  id="components.TabbedItineraries.fareCost"
-                  values={{
-                    hasMaxFare: maxTNCFare && maxTNCFare > minTNCFare,
-                    minTotalFare: (
-                      <FormattedNumber
-                        currency={currency}
-                        currencyDisplay="narrowSymbol"
-                        // This isn't a "real" style prop
-                        // eslint-disable-next-line react/style-prop-object
-                        style="currency"
-                        value={minTotalFare / 100}
-                      />
-                    )
-                  }}
-                />
+                <FormattedMessage id="common.itineraryDescriptions.noTransitFareProvided" />
                 <Bullet />
               </>
+            ) : (
+              minTotalFare > 0 && (
+                <>
+                  <FormattedMessage
+                    id="components.TabbedItineraries.fareCost"
+                    values={{
+                      hasMaxFare: maxTNCFare && maxTNCFare > minTNCFare,
+                      minTotalFare: (
+                        <FormattedNumber
+                          currency={currency}
```

Comment:
Use `fareCurrency` here (see comment above).

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/565/commits/d9c5b8309c89e75580bfaac4e812dbd10f4bda29